templates/advantage/offers/index.html

```diff
@@ -15,9 +15,11 @@ <h1>Your Ubuntu Advantage offers</h1>
     <section class="p-strip u-no-padding--top">
         <div class="row">
             <div class="col-12">
-                <div class="u-no-margin--bottom p-card">
-                    <div class="p-card__content"><span class="p-text--default"><i class="p-icon--spinner u-animation--spin"></i></span>
-                        Loading…</div>
+                <div class="p-card__content">
+                    <span class="p-text--default">
+                    <i class="p-icon--spinner u-animation--spin"></i>
+                    </span>
+                    <p>Loading offers...</p>
```

Comment:
This loading is duplicated

Reply:
What do you mean? 
The idea is that the loading state in the template is the same as the react App's so the mounting of the app is seamless. Do you think it's redundant?

Reply:
I think its good at first but once changed is likely to lead to off jumps and so I would recommend just having it within the JS component.

Reply:
Could you try and put the Loading text next the icon not under?

Reply:
I udpated it, PTAL